bin/keygen/src/lib.rs

```diff
@@ -0,0 +1,6 @@
+//! Keygen crate
+
+pub const BLOCK_PRODUCTION: &str = "block_production";
```

Comment:
```suggestion
pub const BLOCK_PRODUCTION: &str = "block-production";
```

We use dashes in our CLI=)